### DISCUSSION TITLE
Quarkus 1.1.1.Final released - Bugfixes only

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#24410] [Quarkus Blog/Website]
# Quarkus 1.1.1.Final released - Bugfixes only

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-1-1-1-final-released/

**Comment by kabir** [comment]
Test comment - sorry about the spam

**Reply by kabir** [reply]
Sorry, more spam

**Reply by rhusar** [reply]
With `giscus` people can comment directly from the GitHub Discussions tab!
